File: Accomodation_service/WebApplication1/Models/Accomodation.cs
using System;
using System.Collections.Generic;

namespace WebApplication1.Models
{
    public partial class Accomodation
    {
        public int Aid { get; set; }
        public string Name { get; set; } = null!;
        public string Availability { get; set; } = null!;
        public string Address { get; set; } = null!;
        public string Contactno { get; set; } = null!;
    }
}

File: DarshanAppServlets/src/main/java/com/example/demo/entities/Accomodation.java
package com.example.demo.entities;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Entity
@ToString
@Table(name="accomodation")
public class Accomodation {
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "AID")
	int aid;
	@Column
	String name,availability,address,contactno;

}
